packages/web/components/Player/PlayerTile.tsx

```diff
@@ -81,7 +84,7 @@ export const PlayerTile: React.FC<Props> = ({
             <Heading
               size="lg"
               color="white"
-              bgColor="landingGlassDark"
+              // bgColor="landingGlassDark" // This colour doesn't exist in the design system, should be landingDarkGlass
```

Comment:
You don't really need to comment on what is clearly an error, just fix it.

```suggestion
              bgColor="landingDarkGlass"
```

Reply:
I've set the bgColor for the tile heading to whiteAlpha.100 as that seems to fit the design better.

Some screenshots for reference:

**landingDarkGlass**  

![landingDarkGlass Screenshot 2022-12-27 082506](https://user-images.githubusercontent.com/92962228/209596949-dde4ee1a-efe7-46d3-89d0-8481fa6248a6.png)

---

**whiteAlpha.100**  

![no bg Screenshot 2022-12-27 082600](https://user-images.githubusercontent.com/92962228/209597119-5bc229df-9b98-4f85-93e0-6488ebbb97e6.png)